pulumi/infra/postgres.py

```diff
@@ -0,0 +1,115 @@
+from typing import List, Optional
+
+import pulumi_aws as aws
+import pulumi_random as random
+
+import pulumi
+
+
+class Postgres(pulumi.ComponentResource):
+    """
+    A Postgres instance running in RDS.
+    """
+
+    def __init__(
+        self,
+        name: str,
+        vpc_id: pulumi.Input[str],
+        subnet_ids: pulumi.Input[List[str]],
+        nomad_agent_security_group_id: pulumi.Input[str],
+        opts: Optional[pulumi.ResourceOptions] = None,
+    ) -> None:
+        super().__init__("grapl:Postgres", name, None, opts)
+
+        # Burstable, 2GB memory, Gravitron 2.
+        # With 5GB of storage and all-day usage, it comes out to about $24/mo
+        instance_class = "db.t4g.small"
```

Comment:
Can we specify this as a stack configuration value instead?

Reply:
Sure.